### DISCUSSION TITLE
how can I increase dimension between adjacent pages?

### DISCUSSION THREAD

**Discussion by Aliaksandr3by** [sumatrapdfreader/sumatrapdf#2815] [General]
how can I increase dimension between adjacent pages?
and how can I change color between pages? 

![image](https://user-images.githubusercontent.com/11432724/180146031-f4fe8fb2-de8d-428b-8572-2faafb9126ed.png)

**Comment by GitHubRulesOK** [comment]
@Aliaksandr3by 

the background is affected by two settings most obvious is 
`MainWindowBackground = #ff0000`
however that affects other areas so more useful is keep it as it is and
that can then be over-ridden in `FixedPageUI [` section by 	`GradientColors = #0000FF`

and 	`PageSpacing = 4 4` can be increased or decreased in the same section and also in `ComicBookUI [`
